cnv/cnv_users_guide/cnv-collecting-cnv-data.adoc

```diff
@@ -4,13 +4,16 @@ include::modules/cnv-document-attributes.adoc[]
 :context: cnv-collecting-cnv-data
 toc::[]
 
-When opening a support case, it is often helpful to provide debugging
+When opening a support case, it is typically helpful to provide debugging
 information about your cluster to Red Hat Support.
 
 The `must-gather` tool enables you to collect diagnostic information about your
 {product-title} cluster, including virtual machines and other data related to
 {CNVProductName}.
 
+For prompt support, please supply both {product-title} and {CNVProductName}
```

Comment:
~please~ supply diagnostic information for both {product-title} and {CNVProductName}. ~collections~